spec/index.bs

```diff
@@ -772,9 +775,9 @@ Several operations in WebGPU return promises.
 - {{GPUAdapter}}.{{GPUAdapter/requestDevice()}}
 - {{GPUDevice}}.{{GPUDevice/createComputePipelineAsync()}}
 - {{GPUDevice}}.{{GPUDevice/createRenderPipelineAsync()}}
-- {{GPUBuffer}}.{{GPUBuffer/mapAsync()}}
 - {{GPUShaderModule}}.{{GPUShaderModule/getCompilationInfo()}}
 - {{GPUQueue}}.{{GPUQueue/onSubmittedWorkDone()}}
+- {{GPUBuffer}}.{{GPUBuffer/mapAsync()}}
```

Comment:
Mostly I thought it made sense to put getCompilationInfo next to create*PipelineAsync.
I put mapAsync after onSubmittedWorkDone because it's basically a strictly more complicated version of the same thing.